### DISCUSSION TITLE
YT music CAR_RELEASE It is not patchable?

### DISCUSSION THREAD

**Discussion by jclainezr** [ReVanced/revanced-discussions#1118] [Question]
I have tried to patch the latest version of YT music from android automotive 10, but it gives errors when applying the patch and the apk it generates does not work. Does anyone know how I can patch this version?

![music-car-release](https://github.com/user-attachments/assets/03cc3cf6-d5b4-4f2b-85f8-7cc1f92dbb5e)

**Comment by LisoUseInAIKyrios** [comment]
Is your Manager up to date?  It's showing version 7.03.52 is recommended which is not right.

I don't know what the "car release" variant is, and I can't find anything named that on ApkMirror.  If you're trying to use android auto with YT Music then the regular YT Music works and there is no special version needed.

**Reply by jclainezr** [reply]
The manager is up to date. 
The variant is the one installed from the google app store of OS openR link 2.1.2, which is based on android automotive 10. It is the OS of my vehicle. I don't know if it is a special release designed by the owners of the OS. 
I have been able to install the normal patched version of YT music, but I have the problem that it shows me "not available while driving" as soon as the vehicle starts moving, and to change the music I have to stop the car. This don't happen in CAR_RELEASE so I would like to patch it. Also in this variant you cannot listen to music without a premium subscription, maybe it's a completely different version

**Reply by LisoUseInAIKyrios** [reply]
It sounds completely different.

Maybe try turning off location services or denying location for regular YT Music.  Otherwise maybe try using regular YT Music on your phone and connect to your car using Android (If that's possible).

**Reply by jclainezr** [reply]
"turning off location services or denying location for regular YT Music" Is this configured in the app options or in another way?

**Reply by LisoUseInAIKyrios** [reply]
Turning off location services of the device will turn off gps and all location tracking (and presumably "you are moving" detection) for all apps.  It's in the device settings for phones but who knows for Android automotive.
